TLM/TLM/UI/SubTools/LaneConnectorTool.cs

```diff
@@ -13,22 +13,28 @@ namespace TrafficManager.UI.SubTools {
     using UnityEngine;
     using TrafficManager.Util;
     using TrafficManager.UI.Helpers;
+    using TrafficManager.UI.MainMenu;
     using static TrafficManager.Util.Shortcuts;
-    using System;
-    using GenericGameBridge.Service;
 
-    public class LaneConnectorTool : LegacySubTool {
+    public class LaneConnectorTool
+        : LegacySubTool,
```

Comment:
why is this still legacy subtool. is it going to be legacy forever. in that case legacy is a misleading name. I suggest to at least change its documentation.

Reply:
It is still legacy subtool because i tried to do very few changes. If i upgrade this to new subtool style, this would be a totally different and much larger PR.